tox.ini

```diff
@@ -258,7 +258,8 @@ commands=
     pyinstaller -y --clean pyinstaller.spec
 
 [testenv:tarballs]
+basepython = python3
 deps =
 commands =
          python setup.py update_version
-         python setup.py sdist --formats=bztar,gztar,zip bdist_wheel
+         python setup.py sdist --formats=bztar,gztar,zip bdist_wheel --universal
```

Comment:
As a general rule, this kind of release infrastructure improvement should happen outside of the release branch itself.